cranelift/codegen/src/isa/riscv64/inst/mod.rs

```diff
@@ -1530,6 +1533,11 @@ impl Inst {
                 };
                 format!("{} {},{}", v, rd, rm)
             }
+            &MInst::MovFromPReg { rd, rm } => {
+                let rd = format_reg(rd.to_reg(), allocs);
+                let rm = reg_name(Reg::from(rm));
+                format!("fmv.d {},{}", rd, rm)
```

Comment:
Is `fmv.d` the right mnemonic here? That's a floating-point move, no?

Reply:
Good catch, I misread the F64 in the above case for Mov.